crates/re_viewer/data/quick_start_guides/python_native.md

```diff
@@ -0,0 +1,59 @@
+## Python Quick Start
+
+### Installing the Rerun SDK
+
+The Rerun SDK is available on [PyPI](https://pypi.org/) under the
+[`rerun-sdk`](https://pypi.org/project/rerun-sdk/) name. It can be installed like any other
+Python package:
+
+```sh
+pip3 install rerun-sdk
```

Comment:
We usually just write `pip` and `python` everywhere. Let's be consistent. Python3 is in the majority these days, so I think it is safe to drop the 3.

Reply:
I used those _specifically_ for consistency with https://www.rerun.io/docs/getting-started/python

Reply:
I'll fix those and update the docs in another PR.